client/reader/controller.js

```diff
@@ -44,7 +44,7 @@ function renderFeedError( context ) {
 	);
 }
 
-module.exports = {
+export default {
```

Comment:
while we're here, can we switch this to using named exports instead of an object with props?